pkg/controller/util.go

```diff
@@ -721,3 +729,20 @@ func MakeUploadServiceSpec(name string, pvc *v1.PersistentVolumeClaim) *v1.Servi
 	}
 	return service
 }
+
+func deletePod(req podDeleteRequest) error {
+	pod, err := req.podLister.Pods(req.namespace).Get(req.podName)
+	if k8serrors.IsNotFound(err) {
+		return nil
+	}
+	if err == nil && pod.DeletionTimestamp == nil {
+		err = req.k8sClient.CoreV1().Pods(req.namespace).Delete(req.podName, &metav1.DeleteOptions{})
+		if k8serrors.IsNotFound(err) {
+			return nil
+		}
+	}
+	if err != nil {
+		glog.V(1).Infof("error encountered deleting pod (%s): %s", req.podName, err.Error())
```

Comment:
For those that don't check or care about an error here, we log it so there's a record of something going south